src/SDKs/_metadata/servicefabric_resource-manager.txt

```diff
@@ -0,0 +1,11 @@
+2018-05-07 20:57:23 UTC
+
```

Comment:
Please create a `generate.ps1` file similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/generate.ps1) and regenerate the code